synths/core-synths.scd

```diff
@@ -68,13 +68,13 @@ live coding them requires that you have your SuperDirt instance in an environmen
 
 			numFrames = BufFrames.ir(bufnum);
 			windowSize = numFrames * timescale / 29.0;
-			windowSize = windowSize.clip(1000.0*timescale, 4410.0);
+			windowSize = windowSize.clip(1000*timescale, 4410);
```

Comment:
Will that work on all sample rates? If not, replace `4410` by `0.1 * SampleRate.ir`).